### ISSUE TITLE
Make the wait-for-it timeout configurable

### ISSUE THREAD

**Issue by lambada** [CanastaWiki/Canasta-Go#109]
When running `canasta create` the DB container on my server takes a couple of minutes to come good. In the example logs below it took from `21:58:01` until `22:00:12` for the freshly created db to come good.

This is longer than the wait-for-it script allows
https://github.com/CanastaWiki/Canasta-CLI/blob/2fbcb74ab63d09eed057ae113f86b7ea3d047e68/internal/mediawiki/mediawiki.go#L28 (Called by the create CLI command)

This timeout being non-configurable means that the create command bails out, and you can either keep the containers or cleanup. But the problem with keeping the containers/config is that there doesn't seem to be a way to re-run the create command with the same config so that the install command etc can be re-run now the DB is up. ie the following code doesn't appear to be able to be re-run https://github.com/CanastaWiki/Canasta-CLI/blob/2fbcb74ab63d09eed057ae113f86b7ea3d047e68/cmd/create/create.go#L122-L130

Instead, rerunning create with the same args gives the following fatal error
```
fatal: destination path '/canasta/canasta1' already exists and is not an empty directory.
```

Allowing the timeout to be configurable would probably be an easier solution to this than attempting to rerun a partially-completed installation process.

```
$ docker logs canasta1-db-1
2024-04-28 21:58:01+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.0.33-1.el8 started.
<snip>
2024-04-28T22:00:12.657246Z 0 [System] [MY-011323] [Server] X Plugin ready for connections. Bind-address: '::' port: 33060, socket: /var/run/mysqld/mysqlx.sock
2024-04-28T22:00:12.657287Z 0 [System] [MY-010931] [Server] /usr/sbin/mysqld: ready for connections. Version: '8.0.33'  socket: '/var/lib/mysql/mysql.sock'  port: 3306  MySQL Community Server - GPL.
```

**Comment by jeffw16** [comment]
I agree that this is an issue that we should aim to resolve. @vedmaka do you think it would be feasible to allow the sysadmin to change `wait-for-it.sh` depending on how fast/slow the system is?

**Comment by vedmaka** [comment]
While the wait timeout can be exposed through an ENV pretty easily, I do not think it's a good idea

The reason is that instead, there must be some external readiness check in place ( i.e. a health check on the database container and [service_healthy](https://docs.docker.com/compose/how-tos/startup-order/) that ensures that the wiki container runs only after the dependency database container is healthy & ready

The embedded wait-for-it check can serve as a tiny compensation for any random delay in connectivity, but should not be more than that

**Comment by lambada** [comment]
Mmm, fair point. In that case the Canasta Docker Compose project should configure appropriate readiness checks
